views/includes/check-your-answers/beneficial-owner-gov.html

```diff
@@ -80,7 +80,11 @@
 {% endset %}
 
 {% if pageParams.isRegistration %}
-  {% set changeLinkUrl = OE_CONFIGS.BENEFICIAL_OWNER_GOV_URL + "/" + bog.id %}
+  {% if not IS_REDIS_REMOVAL_ENABLED %}
```

Comment:
I suggest simplifying the logic - remove the `not` and switch around what the blocks do.

Reply:
The `not` ensures that the larger chunk of the `if` statement is at the bottom (in the `else` clause) and not the top, which makes for cleaner, better formatted code (and avoids having top-heavy conditionals).

Reply:
When you say 'larger' you mean that in the `else` block the line is a bit longer in length?

Not sure that justifies the addition of `not`, but up to you.